docs/modules/ROOT/pages/references/service-maturity.adoc

```diff
@@ -0,0 +1,142 @@
+= Service Maturity
+
+TIP: The intended audience of this document is the Product Owner of the service. It specifies a high-level set of functionality which helps in navigating through the implementation phase. Breaking these iterations down into technical details is part of the engineering process.
+
+Implementing and maintaining a Managed Kubernetes service is iterative work.
+The idea is to release early and release often, without having the urge to have all iterations available right from the beginning.
+
+This page gives an idea on the iterative steps and a way to assess the Managed Kubernetes service maturity.
+Except the "Cluster Provisioning" iteration, all steps in-between can be in any order, as it fits best.
+When all iterations are implemented, it's not meant that the service doesn't receive new functionality or doesn't get maintained anymore, we're talking about the default scope outlined on this page.
+
+Important is that every step is independent of each other. A Managed Kubernetes service that implements _Cluster Maintenance_ doesn't necessarily already implement _Logs_.
+That in turn also means we must allow some leeway for each Managed Kubernetes service in respect to the requirements, should a service not allow for a feature or makes it overly complex.
+A Managed Kubernetes service is allowed to deviate from a requirement when it makes sense and can be documented and reasoned about accordingly.
+
+To understand the personas referenced in this document, see xref:references/glossary.adoc#_personas[Glossary - Personas].
+
+== Initial Release: Cluster Provisioning
+
+As an *End-User* I can:
+
+* provision a Cloud Kubernetes environment on our supported Cloud Service Providers, fully self-service.
+* connect to the Cloud Kubernetes API for using it.
+* customize my cluster with a pre-defined set of features and components.
+** scale my cluster horizontally and vertically, as my needs change.
+** choose the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* delete the Cloud Kubernetes environment again.
+
+As a *Technical VSHNeer* I can:
+
+* see and configure the https://products.docs.vshn.ch/products/service_levels.html[service level^] of the instance.
+* automatically bill the service to the end-user according to the specified price model.
```

Comment:
Maybe that is a role on its own? A _Technical VSHNeer_ actually does not necessarily care about billing (at least not directly). It is actually VSHN as a whole that needs this.

Reply:
You're right. Coming up with another Persona is something I'd like to invest a bit more time and for the time being, leave it as it is. Once we fine-tuned to personas, we can adapt it here as well.